corehq/apps/hqwebapp/static/hqwebapp/js/widgets.js

```diff
@@ -98,6 +99,8 @@ hqDefine("hqwebapp/js/widgets",[
                     geocoder.setInput(getGeocoderValue());
                 }
             });
+
+            $('.date-picker').datepicker({ dateFormat: "yy-mm-dd" });
```

Comment:
I think this should be 12 space.

Reply:
yeah, I don't know why stickler is complaining here

Reply:
My IDE thinks the whole function is indented 4 spaces too many - that's probably what it's reading here.